photo/types.py

```diff
@@ -15,6 +16,7 @@
 
 @strawberry.django.type(User)
 class UserType:
+    id: uuid.UUID
```

Comment:
this will conflict with the changes I made in the UserType. @kenvontucky should this be a string or uuid?

Reply:
uuid.UUID will behave as string when queried.